lib/std/os/uefi/protocols/device_path_protocol.zig

```diff
@@ -491,19 +799,43 @@ pub const MessagingDevicePath = union(Subtype) {
         type: DevicePathType,
         subtype: Subtype,
         length: u16,
-        reserved: u16,
-        baud_rate: u32,
+        reserved: u32,
+        baud_rate: u64,
```

Comment:
This doesn't fix the problem. You've just made that accidental padding explicit. Use a similar solution as the above.

Reply:
This is actually correct, UEFI defines `reserved` as having 4 bytes and `baud_rate` as being 8 bytes. I'm not sure why these were 2 and 4. Check `Table 10-34 UART Device Path`

Reply:
Fair enough, was comparing against the original (and assuming it was correct).